docs/canvas/canvas-function-reference.asciidoc

```diff
@@ -1,17 +1,2133 @@
+-------------
+
+*Returns:* `image`
+
+
+[float]
+[[location_fn]]
+=== `location`
+
+Uses the browser's location functionality to get your current location. Usually 
+quite slow, but fairly accurate.
+
+*Accepts:* `null`
+
+*Returns:* `datatable`
+
+
+[float]
+[[l_fns]]
+== L
+[[lt_fn]]
+=== `lt`
+
+Returns whether the _context_ is less than the argument passed in.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[lte_fn]]
+=== `lte`
+
+Returns whether the _context_ is less than or equal to the argument passed in.
+
+*Accepts:* `number`, `string`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `value`
+|`number`, `string`
+|The value to compare the _context_ to
+|===
+
+*Returns:* `boolean`
+
+
+[float]
+[[m_fns]]
+== M
+[[mapColumn_fn]]
+=== `mapColumn`
+
+Adds a column calculated as the result of other columns, or not. See also <<alterColumn_fn>> and <<staticColumn_fn>>.
+
+*Accepts:* `datatable`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *** 
+
+Alias: `column`
+|`string`
+|The name of the resulting column
+
+|`expression` ***
+
+Alias: `exp`, `fn`, `function`
+|`boolean`, `number`, `string`, `null`
+|A Canvas expression that will be passed to each row as a single row `datatable`
+|===
+
+*Returns:* `datatable`
+
+
+[float]
+[[markdown_fn]]
+=== `markdown`
+
+Adds an element for rendering Markdown text. Great for single numbers, metrics, or paragraphs of text.
+
+*Accepts:* `datatable`, `null`
+
+[cols="3*^<"]
+|===
+|Argument |Type |Description
+
+|_Unnamed_ *†*
+
+Alias: `expression`, `content`
+|`string`
+|A `string` containing Markdown. You can pass this multiple 
+times to achieve concatenation 
+
+Default: `""`
+
+|`font`
+|`style`
+|Font settings. Technically, you can put other styles in here too 
```

Comment:
What kinds of other styles? And what does `technically` refer to?

Reply:
In theory, other CSS styles could be applied to the `style` object, but I think in practice, it's not possible to pass any styles that aren't defined as arguments in the `font` function. I'm in favor of removing the sentence about other styles because the function is written to only accept font related CSS styles.

Reply:
Let's remove it! To make the first part a complete sentence, I would simply change it to:

`The font settings.`